src/main/java/com/flightstats/hub/model/ChannelConfig.java

```diff
@@ -93,15 +93,21 @@ private ChannelConfig(String name,
             this.ttlDays = ttlDays;
             this.maxItems = maxItems;
         }
-
         if (isBlank(storage)) {
-            this.storage = SINGLE.name();
+            this.storage = BATCH.name();
         } else {
             this.storage = StringUtils.upperCase(storage);
```

Comment:
What happens if we put some garbage in `storage`? Is that validated elsewhere?

Reply:
I think we do have validation elsewhere. I'm wondering if that exception that we throw could be moved there. I'll try to figure out where we do that. One of the services, maybe?

Reply:
There's a `ChannelValidator` class that might be the right place to put that sort of thing... and it does check for a valid storage type.